roles/ansible-tdp-common-actions/tasks/deploy-ranger-hbase-policy.yml

```
@@ -0,0 +1,30 @@
+---
+- name: "Ranger: get policy list"
+  uri:
+    url: "{{ ranger_hdfs_install_properties.POLICY_MGR_URL }}/service/public/v2/api/policy"
```

```suggestion
    url: "{{ ranger_hbase_install_properties.POLICY_MGR_URL }}/service/public/v2/api/policy"
```

You should use hbase property for this

---

Thanks for reviewing!

Using hdfs, hive or hbase properties doesn't seem to change the result, as the information each return seems to be exactly the same for all services properties (all ranger-policies from all services, as ids are for all of them). Not sure if this is intended to work like this.   

I will still change it to hbase for clarity if you think it's ok, and I'll add the same thing in the hive-ranger policies as it also uses the hdfs properties.